ember-toucan-core/src/components/form/radio-field.ts

```diff
@@ -0,0 +1,40 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+import type { ToucanFormRadioControlComponentSignature } from './controls/radio';
+
+export interface ToucanFormRadioFieldComponentSignature {
+  Element: HTMLInputElement;
+  Args: {
+    hint?: string;
+    isDisabled?: boolean;
+    label: string;
+    name: string;
+    onChange?: ToucanFormRadioControlComponentSignature['Args']['onChange'];
+    /**
+     * A test selector for targeting the root element of the field. In this case, the wrapping div element.
+     */
+    rootTestSelector?: string;
+    /**
+     * This component argument is used to determine if the underlying radio is checked.
+     * When `selectedValue` and `value` are equal, the radio will have the checked attribute applied.
+     */
+    selectedValue?: string;
```

Comment:
It would be pretty much pointless to try to disagree here, given that this is _exactly_ what I am doing in headless [here](https://github.com/CrowdStrike/ember-headless-form/blob/main/packages/ember-headless-form/src/-private/components/control/radio.ts#L25) and [here](https://github.com/CrowdStrike/ember-headless-form/blob/main/packages/ember-headless-form/src/-private/components/control/radio.ts#L56-L58) 😅

Reply:
I love it.  I'm going to do this as part of a follow up PR as this PR is blowing up quite a bit anyway.  Update coming soon!